components/match2/wikis/counterstrike/match_group_input_custom.lua

```diff
@@ -253,7 +249,7 @@ end
 function MatchFunctions.getExtraData(match, opponents)
 	return {
 		mapveto = MatchGroupInputUtil.getMapVeto(match),
-		status = MatchFunctions.getMatchStatus(match),
+		status = match.resulttype == MatchGroupInputUtil.RESULT_TYPE.NOT_PLAYED and match.status or nil,
```

Comment:
```suggestion
		status = match.resulttype == MatchGroupInputUtil.RESULT_TYPE.NOT_PLAYED and Logic.emptyOr(match.status, Variables.varDefault('tournament_status') or nil,
```
Would also be needed in that case
Same preference with match.status input removed

Reply:
doesn't work if input is done via finished param